Library/Homebrew/cask/cmd.rb

```diff
@@ -71,7 +71,7 @@ class Cmd
       Cmd::Style     => "brew style",
       Cmd::Uninstall => "brew uninstall [--cask]",
       Cmd::Upgrade   => "brew upgrade [--cask]",
-      Cmd::Zap       => "brew upgrade --zap [--cask]",
+      Cmd::Zap       => "brew uninstall --zap [--cask]",
```

Comment:
@MikeMcQuaid is this still how this should look since `--zap` is exclusive to cask? `--cask` would (or should) always be present if `--zap` is right?

Reply:
`--zap` is exclusive to cask but `--cask` is still optional even when specifying a cask.